app/checkout/[userId]/page.tsx

```diff
@@ -56,6 +57,24 @@ export default function CheckOut({ params }: CheckOutProps) {
         router.push(`/`);
     };
 
+    const onChange = (visitData: Visit) => {
+        setVisitFormData(visitData);
+    };
+
+    const saveVisit = async (visitData: Visit) => {
+        await updateVisit(visitData, params.userId);
+        const client = await updateClient({
+            ...clientData,
+        });
```

Comment:
is updating clientData needed?

Reply:
Saving the visit won't mutate client's profile, so I'm removing it from here.